_books/ion-1-1/src/text/values.md

```diff
@@ -0,0 +1,489 @@
+# Values
+
+## Annotations
+In the text format, type annotations are denoted by a [symbol token](symbol-tokens.md) and double-colons preceding any value.
```

Comment:
We should probably note that e-expressions and expression groups are not values, and cannot be annotated.

Reply:
I can mention it, but it's not specific to the text encoding. I tried to keep this mostly focused to the encoding itself.